ex3/assignment1_matchismo/PlayingCardGameViewController.m

```diff
@@ -0,0 +1,64 @@
+//
+//  PlayingCardGameViewController.m
+//  assignment1_matchismo
+//
+//  Created by Omer Plotnik on 11/08/2020.
+//  Copyright © 2020 lightricks. All rights reserved.
+//
+
+#import "PlayingCardGameViewController.h"
+#import <Foundation/Foundation.h>
+#import "CardMatchingGame.h"
+#import "PlayingCardDeck.h"
+
+
+@interface PlayingCardGameViewController()
+
+@property (weak, nonatomic) IBOutlet UILabel *scoreLabel;
+
+@property (weak, nonatomic) IBOutlet UILabel *activityLabel;
+
+@end
+
+@implementation PlayingCardGameViewController
+
+-(Deck *)createDeck
+{
+    return [[PlayingCardDeck alloc] init];
+}
+
+
+- (void)updateUI
+{
+    for (UIButton *cardButton in self.cardButtons) {
+        NSInteger cardButtonIndex = [self.cardButtons indexOfObject:cardButton];
+        Card *card = [self.game cardAtIndex:cardButtonIndex];
+        [cardButton setAttributedTitle:[self titleForCard:card] forState:UIControlStateNormal];
+        [cardButton setBackgroundImage:[self backgroundImageForCard:card] forState:UIControlStateNormal];
+        cardButton.enabled = !card.isMatched;
+    }
+  [super updateUI];
```

Comment:
Can this be at the first line of the function?

Reply:
Done! (also in SetCardGameViewController)